packages/api-fetch/src/test/index.js

```diff
@@ -127,6 +127,7 @@ describe( 'apiFetch', () => {
 		);
 
 		return apiFetch( { path: '/random' } ).catch( ( body ) => {
+			// eslint-disable-next-line jest/no-conditional-expect
```

Comment:
The `jest/no-condtional-expect` rule now also covers catching promises, so I had to add these comments to disable the rule.

Reply:
Link: https://github.com/jest-community/eslint-plugin-jest/pull/819